streamz/tests/test_core.py

```diff
@@ -1063,5 +1064,23 @@ def test_share_common_ioloop(clean):
     assert aa.loop is bb.loop
 
 
+@pytest.mark.parametrize('delimiter', ['\n', '=='])
+def test_from_text(delimiter):
+    with tmpfile() as fn:
+        open(fn, 'w')  # touch file - needs to exist
+        stream = Stream.from_textfile(fn, delimiter=delimiter,
+                                      poll_interval=0.01)
+        l = stream.sink_to_list()
```

Comment:
Slight preference to use the pronoun `L` rather than `l` for an anonymous list.  This matches style in the rest of the codebase and, I think, Python programs generally (though this could just be what I'm used to).

Reply:
pycharm says "function variable should be lower case" :)  I'm happy either way.

Reply:
Harumph